src/ledger.rs

```diff
@@ -550,7 +550,7 @@ mod tests {
 
     fn tmp_ledger_path(name: &str) -> String {
         use std::env;
-        let out_dir = env::var("OUT_DIR").unwrap();
+        let out_dir = env::var("OUT_DIR").unwrap_or_else(|_| "target".to_string());
```

Comment:
should be a function `get_out_dir() -> String` which can share this logic?

Reply:
yeah, eventually I'd like to collapse temp_dir stuff to a single implementation.  I had to hurry up and throw this in because for some reason cargo stopped offering "OUT_DIR" in some of my test runs (like cargo watch -x test, maybe)...